code-challenges.test.js

```diff
@@ -0,0 +1,152 @@
+// ASSESSMENT 3: Coding Practical Questions with Jest
+
+// Please read all questions thoroughly
+// Pseudo coding is REQUIRED
+// If you get stuck, please leave comments to help us understand your thought process
+
+// Use test driven development to complete the following questions
+// Add appropriate dependencies: $ yarn add jest
+
+// Reminder: The test will call your function
+// Run the file with the following command: $ yarn jest
+
+// --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+
+// a) Create a test with expect statements for each of the variables provided.
+describe("fibonacci", () => {})
+it("returns an array length containing the numbers of the Fibonacci sequence", () => {
+    expect(fibonacci(fibLengthOne)).toEqual([1,1, 2, 3, 5, 8])
+    expect(fibonacci(fibLengthTwo)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+})
+
+const fibLengthOne = 6
+// Expected output: [1, 1, 2, 3, 5, 8]
+
+const fibLengthTwo = 10
+// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+// b) Create the function that makes the test pass.
+// Pseudocode: First create a function called fibonacci that will take in a number. Then create a variable returnArray which will hold the numbers. Then i will use the for loop that will iterate the returnArray starting at the second index and start the number input in increments starting at one.
+
+const fibonacci = (number) => {
+    let returnArray = [1, 1]
+    for (let i = 2; returnArray.length < number; i++) {
+      returnArray.push(returnArray[i - 1] + returnArray[i - 2])
```

Comment:
Nice logic you have going on here! 👏🏽 One of the things that will help your pseudocode be more descriptive is talking about what you are using push for and the reason for i-1 and -2.